Fix recursive make calls on G/FreeBSD
Merged upstream; will appear in 4.82
Or in:
ftp://lsof.itap.purdue.edu/pub/tools/unix/lsof/NEW/lsof_4.82B.freebsd.tar.bz2

https://bugs.gentoo.org/show_bug.cgi?id=250383

--- lsof_4.81/lsof_4.81_src/dialects/freebsd/Makefile.old	2008-12-09 12:56:40 +0000
+++ lsof_4.81/lsof_4.81_src/dialects/freebsd/Makefile	2008-12-09 12:56:48 +0000
@@ -129,7 +129,7 @@
 
 dnode2.o:	dzfs.h dnode2.c
 		@if [ -f ./Makefile.zfs ]; then \
-		    make -f Makefile.zfs dnode2.o; \
+		    ${MAKE} -f Makefile.zfs dnode2.o; \
 		else \
 		    echo "${CC} ${CFLAGS} -c dnode2.c"; \
 		    ${CC} ${CFLAGS} -c dnode2.c; \
